src/main/java/org/justinbaur/bankteller/service/UserProfileServiceDatabaseImpl.java

```diff
@@ -0,0 +1,63 @@
+package org.justinbaur.bankteller.service;
+
+import org.justinbaur.bankteller.domain.Profile;
+import org.justinbaur.bankteller.exceptions.AccountNotFound;
+import org.justinbaur.bankteller.exceptions.ProfileNotFound;
+import org.justinbaur.bankteller.exceptions.UpdateException;
+import org.justinbaur.bankteller.repository.ProfileRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserProfileServiceDatabaseImpl extends ProfileServiceImpl implements UserProfileService {
```

Comment:
Again, why extend this?

Reply:
Same reason as above, am I misunderstanding something?